src/classes/STG_PanelRDBatch_CTRL.cls

```diff
@@ -51,6 +51,10 @@ public with sharing class STG_PanelRDBatch_CTRL extends STG_Panel {
     * @description Whether the batch process is currently running  
     */
     public boolean isRunningBatch { get; set; }
+
+    /* @description Batch size of Recurring Donations Apex job  
+    */
+    public Integer batchSize = (UTIL_CustomSettingsFacade.getRecurringDonationsSettings().Recurring_Donation_Batch_Size__c <> null) ? UTIL_CustomSettingsFacade.getRecurringDonationsSettings().Recurring_Donation_Batch_Size__c.intValue() : 50;
```

Comment:
Rather than recreate the var here (repeating the code), it's probably better to do the following in the `runBatch()` method below:

```
RD_RecurringDonations_BATCH rdb = new RD_RecurringDonations_BATCH();
Id batchInstanceId = Database.executeBatch(rdb, rdb.batchSize);
```

Reply:
Totally. Good call. I'll make that change now.